docs/traceflow-guide.md

```diff
@@ -110,7 +119,54 @@ spec:
 The CRD above starts a new trace from source Pod named `tcp-sts-0` to destination Pod named `tcp-sts-2` using ICMPv6
 protocol.
 
-### Using antctl and spec config
+### Live-traffic Traceflow
+
+Starting from Antrea version 1.0.0, you can trace a packet of the real traffic
+from or to a Pod, instead of the injected packet. To start such a Traceflow, add
+`liveTraffic: true` to the Traceflow `spec`. Then the first packet of the first
+connection that matches the Traceflow spec will be traced, and the headers of
+the packet will captured and reported to the `status` field of the Traceflow
+CRD, besides the observations. A live-traffic Traceflow requires only one of
+`source` and `destination` to be specified. When `source` or `destination` is
+not specified, it means to capture a packet from any source or to any
+destination. One of `source` and `destination`  must be a Pod. When `source` is
+not specified, or is an IP address, only the receiver Node will capture the
+packet and trace it after the L2 forwarding observation point. This means even
+the source of the packet is on the same Node as the destination, no observations
+on the sending path will be reported for the Traceflow. By default, a
+live-traffic Traceflow (the same as a normal Traceflow) will timeout in 15
```

Comment:
I think we have default 20s time out  according to https://github.com/vmware-tanzu/antrea/blob/3335e734071894f70a9ab33c734799f56c049689/pkg/apis/crd/v1alpha1/types.go#L97?
and looks like we have different timeout setting in antctl and traceflow controller, should we make them be consistent?

Reply:
You are right. Updated to 20 seconds.

I think for antctl we want to return faster so used shorter timeout.